packages/kubernetes-1.16/kubelet-config

```diff
@@ -74,6 +74,10 @@ systemReserved:
   {{@key}}: "{{this}}"
   {{~/each}}
 {{~/if}}
+cpuManagerPolicy: {{default "none" settings.kubernetes.cpu-manager-policy}}
```

Comment:
Would you please confirm how you tested this on 1.16 - 1.19?  We weren't listing the setting before, so there's potential for kubelet to behave differently now that we're giving it an explicit value, even if we think it's the same.

Reply:
Yeah, sure thing.

Reply:
### Test for 1.16 -1.19 variants
 check if `cpu-manager-policy` has been configured expectantly and test if fluent-bit pod work as expected:

Test with `cpu-manager-policy= "none"`
```
1726 policy_none.go:43] [cpumanager] none policy: Start
```
Test with default ("none")
```
1737 cpu_manager.go:193] [cpumanager] starting with none policy
```
Test with `cpu-manager-policy= "static"`
```
1723 cpu_manager.go:193] [cpumanager] starting with static policy
```

configure it as `none` and test if fluent-bit pod work as expected:
1. kubectl run --image public.ecr.aws/aws-observability/aws-for-fluent-bit:2.16.0 --command -- sh
2. kubectl exec -ti sh -- sh
3. /fluent-bit/bin/fluent-bit -H -i cpu -o null 
4. while curl localhost:2020; do echo pass; done

Migration test - check if `cpu-manager-policy` still be started after upgrade/downgrade even unusual checkpoint value (same as test for 1.20).

Reply:
We also confirmed offline that regardless of whether you have an explicit "none" setting or the default, you do see both messages, `none policy: Start` and `starting with none policy` - I wanted to make sure we weren't taking a different code path.